tests/helpers.py

```diff
@@ -63,20 +63,27 @@ def user_login(client, username, users):
     user_logout(client)
     if username != "anonymous":
         user = User.query.get(users[username].id)
-        # needed for sessions/http requests
-        login_user_via_session(client, user)
         # needed for Identity/Permissions loading
-        login_user(user)
+        login_user(user, remember=True)
+        # needed for sessions/http requests
+        login_user_via_session(client, email=user.email)
+
         return user
 
 
 def user_logout(client):
     """Util function to log out user."""
     with client.session_transaction() as sess:
+        from flask import g
```

Comment:
You might want to get rid of all these ones, and use what is already provided in [pytest-invenio](https://github.com/inveniosoftware/pytest-invenio/blob/master/pytest_invenio/user.py).

Reply:
I would clean it up in next opportunity, since it would require going through all the tests and updating how the user fixture is used. I think this PR is already sizeable.